packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -46,6 +46,8 @@ package.json
 | --externalPort                   | number  |                                                  | The port mapping from the host to the container.                                                                                          |
 | --formFactors                    | array   |                                                  | A list of form factors for the test run. One of tiny, small, medium, large, huge, or enormous                                             |
 | --gridUrl                        | string  |                                                  | The remote selenium grid address.                                                                                                         |
+| --seleniumServiceUrl             | string  |                                                  | The address for the selenium service.                                                                                                     |
+| --port                           | number  |                                                  | The port mapping for the selenium service or the external selenium grid.                                                                  |
```

Comment:
add 4444 as the default value (yeah it's more complicated than that but hard to explain in a little place.)